presto-main/src/main/java/com/facebook/presto/server/QueryStateInfo.java

```diff
@@ -41,7 +41,7 @@
     private final Optional<String> clientInfo;
     private final Optional<String> catalog;
     private final Optional<String> schema;
-    private final Optional<List<ResourceGroupInfo>> resourceGroupChain;
+    private final Optional<List<ResourceGroupInfo>> pathToRoot;
```

Comment:
Might as well merge this diff with the first one? I don't have strong opinion though.